adaptive_scheduler/kernel_mappings.py

```diff
@@ -130,8 +131,14 @@ def get_rise_set_timepoint_intervals(rise_set_target, visibility, max_airmass, m
     '''
     # arguments are packed into a tuple since multiprocessing pools only work with single arg functions
     rs_dark_intervals = visibility.get_dark_intervals()
-    rs_up_intervals = visibility.get_target_intervals(target=rise_set_target, up=True,
-                                                      airmass=max_airmass)
+    rs_up_intervals = []
```

Comment:
this part is just a local bug i fixed in the existing scheduler that needed to get added to this one. It just ignores targets that throw an exception when using rise-set